feature/matcher/src/main/java/com/simprints/matcher/usecases/FaceMatcherUseCase.kt

```diff
@@ -41,7 +41,11 @@ internal class FaceMatcherUseCase @Inject constructor(
         project: Project,
     ): Flow<MatcherState> = channelFlow {
         Simber.i("Initialising matcher", tag = crashReportTag)
-        val bioSdk = resolveFaceBioSdk()
+        if (matchParams.faceSDK == null) {
+            send(MatcherState.Success(emptyList(), 0, ""))
```

Comment:
Should we log a Simber.e() here (and in fingerprint's copy) as this would indicate a code error somewhere?

Reply:
I will add `Simber.w` since there is no exception to log with `.e()`.